subsys/bluetooth/controller/ll_sw/coex_ticker.c

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+
+#include <zephyr.h>
+#include <device.h>
+#include <init.h>
+#include <soc.h>
+
+#include <logging/log.h>
+#include <drivers/gpio.h>
+#include <sys/__assert.h>
+#include <bluetooth/hci.h>
+#include "controller/hal/ticker.h"
```

Comment:
blank line to separate the controller includes.

```suggestion
#include <bluetooth/hci.h>

#include "controller/hal/ticker.h"
```